docs/map.xsd

```diff
@@ -17,24 +17,96 @@
   types
 -->
 
+  <xs:simpleType name="pixelT">
+    <xs:restriction base="xs:nonNegativeInteger"></xs:restriction>
+  </xs:simpleType>
+
 <xs:simpleType name="encodingT">
   <xs:restriction base="xs:NMTOKEN">
     <xs:enumeration value="base64" />
+    <xs:enumeration value="csv" />
   </xs:restriction>
 </xs:simpleType>
 
+  <xs:simpleType name="formatorsourceT">
```

Comment:
Ah, a type I tried to add to create a choice between format or source. I'll remove it. Ended up choosing 'optional' as you suggested